nextjs-task-system/app/api/tasks/route.ts

```
@@ -0,0 +1,210 @@
+  const url = new URL(req.url);
+  const userIdOrGroupId = url.searchParams.get("userIdOrGroupId");
+  const status = url.searchParams.get("status");
+  const priority = url.searchParams.get("priority");
+  const sortBy = url.searchParams.get("sortBy") || "createdAt";
+  const direction = url.searchParams.get("direction") || "desc";
+
+  // validate userId
+  const [type, id] = userIdOrGroupId?.split(",") || ["", ""];
+  const userIdInt = type === "user" && id ? parseInt(id, 10) : undefined;
+  const groupIdInt = type === "group" && id ? parseInt(id, 10) : undefined;
+  if (type === "user" && isNaN(parseInt(id))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)